src/__test__/utils/sendWork.test.js

```diff
@@ -1,6 +1,6 @@
 import { v4 as uuidv4 } from 'uuid';
```

Comment:
Similarly, I would rename this test file to be called `seekFromAPI.test.js` and I will put it in that `work` folder in `__test__/utils`

Reply:
now fixed